documentation/depmap_omics_blogpost.md

```diff
@@ -41,80 +47,87 @@ If you are not familiar with these notions, we will first recommend you get more
 
-Go to the repos and pull them to the same parent folder as `ccle_processing`.
+```bash
+git clone https://github.com/broadinstitute/ccle_processing.git
+git clone https://github.com/jkobject/JKBio.git
+git clone https://github.com/broadinstitute/gkugener.git
+```
 
 
-### :warning: you would need the approriate R packages and python packages
+### :warning: you would need the approriate R packages and python packages -- what are the appropriate de
 
-1. You will need to install jupyter notebooks and google cloud sdk
+1. You will need to install [Jupyter Notebook](https://jupyter.org/install) and Google Cloud SDK
   - install [Google Cloud SDK](https://cloud.google.com/sdk/docs/downloads-interactive).
-  - authenticate my SDK account by running `gcloud auth application-default login` in the terminal.
+  - authenticate your Google account by running `gcloud auth application-default login` in the terminal.
 2. For R packages, a loading function contains all required ones (in [here](https://github.com/broadinstitute/gkugener/blob/master/RScripts/load_libraries_and_annotations.R))
-  - Another R package needs to be installed like so: `cd src/cdsomics && R CMD INSTALL . && cd -`.
-  - Also taigr: `cd ../JKBio/taigr && R CMD INSTALL . && cd -`.
-  - And Celllinemapr: `cd ../JKBio/cell_line_mapping-master/celllinemapr && R CMD INSTALL . && cd -`.
-3. For Python use the requirements.txt file `pip install -r requirements.txt`.
+install the following R packages using the provided commands:
+  - install `cdsomics`: `cd src/cdsomics && R CMD INSTALL . && cd -`.
+  - inst `taigr`: `cd ../JKBio/taigr && R CMD INSTALL . && cd -`.
+  - And `Celllinemapr`: `cd ../JKBio/cell_line_mapping-master/celllinemapr && R CMD INSTALL . && cd -`.
+3. Install the Python dependencies: `pip install -r requirements.txt`.
 
 
 ### Creating your Terra Workspaces:
 
-use the `data/xQx/.json` which lists the parameters used for each workflows of each off the 3 workspaces in our pipeline (the CSV lists the workflows with their correct name):
+use the `data/xQx/.json` which lists the parameters used for each workflow of each off the 3 workspaces in our pipelines (the CSV lists the workflows with their correct name): 
 - import the workflows, with their parameters listed in here.
 - import the workspace parameters/data listed in the `GENERAL` field.
 
 Once you have set up your workspace with the corresponding workflows, workspace data and workflow input/output parameters, you can import your data to be processed. __Tools available in `TerraFunction` in the `JKBio` package as well as the `dalmatian` package can be used to automate this process.__
 
-Once this is done, you can run your jupyter notebook server and open one of the 3 `CCLE_\*` jupyter files corresponding to our 3 pipelines.
+Once this is done, you can run your Jupyter Notebook server and open one of the 3 `CCLE_\*` Jupyter files corresponding to our 3 pipelines. These three pipelines are ...
 
```

Comment:
I put in some incomplete sentences. Complete them on your own. Explain what these pipelines are.